README.md

```diff
@@ -110,6 +110,7 @@ environment variable:
 
 ```yaml
 password: { fromEnv: ENVIRONMENT_VARIABLE }
+application_credential_secret: { fromEnv: ENVIRONMENT_VARIABLE }
```

Comment:
Please make that into a full example (and also explain `application_credential_{id,name}`), separate from the username/password-based examples, and link to the full config file like the other examples do.

Reply:
done